gke/variables.tf

```
@@ -106,5 +112,11 @@ variable "nomad_count" {
 variable "nomad_ssh_enabled" {
   type        = bool
   default     = false
-  description = "Enables SSH to Nomad clients. If enabled, use `gcloud compute ssh` to manage SSH keys"
+  description = "Enables SSH to Nomad clients. If enabled, use `gcloud compute ssh` to manage SSH keys. If you use a bastion host and a private endpoint, you can still connect to Nomad clients via the bastion host using their private IPs"
```

I _think_ this is the case?
```suggestion
  description = "Enables SSH to Nomad clients. If enabled, use `gcloud compute ssh` to manage SSH keys. If you use a bastion host and a private endpoint, you can still connect to Nomad clients with this value set to `false` via the bastion host using their private IPs"
```